### ISSUE TITLE
Add County Domain Values

### ISSUE THREAD

**Issue by geofinne** [NENA911/NG911GISDataModel#47]
Add the list of County values into the domain.  

Per the standard, the domain should be:  
_Restricted to the names of counties and county equivalents. For the US, a complete list is maintained by the US Census Bureau as ANSI INCITS 31:2009 [17] (Formerly FIPS 6-4) and the Domain is restricted to the exact listed values as published in ANSI INCITS 31:2009 [17], including casing and use of abbreviations._

**Comment by tomneer** [comment]
Adding County and Equivalents would create a domain containing several thousand records. General practice is for the GIS Data Provider to customize this domain to their organizational county(s) and surrounding counties. Documentation has been added about [GIS Data Provider Specific Domains](https://github.com/NENA911/NG911GISDataModel#gis-data-provider-specific-domains), their population and management. 

However, if you feel this is something to discussion with the Data Model Template WG, we can add it to our next agenda.

**Comment by geofinne** [comment]
I think it's worth a discussion.  If we don't add the values, it opens up the possibility for misspellings and incorrect formatting (e.g., "King" instead of the required "King County" format).

**Comment by jasonhorning** [comment]
We did discuss this in V1 and chose not to add the 3000+ US county values, and just left it up to the users to populate for their state or region.  We did add this documentation to our readme in V1.

_Domain Use
Within the templates there are some domains that have no entries.  These domains include AdditionalCode, AgencyID, County, ESN, PostalCode, PostalCommunityName and ServiceURI.  The entity is expected to populate these domains, in accordance with guidelines specified within the NG9-1-1 GIS Data Model standard, based on the needs within their jurisdiction._

Consider that we'd also have to add all of the same values for MX and CA so there would likely be far more than 3k.

If you'd like to still have a conversation about it we can but that's the background.
